onebusaway-android/src/main/java/org/onebusaway/android/util/RegionUtils.java

```diff
@@ -444,11 +445,12 @@ public static String formatOtpBaseUrl(String baseUrl) {
                         c.getInt(6) > 0,            // Supports Oba Discovery
                         c.getInt(7) > 0,            // Supports Oba Realtime
                         c.getInt(8) > 0,            // Supports Siri Realtime
-                        c.getString(9),              // Twitter URL
-                        c.getInt(10) > 0,            // Experimental
-                        c.getString(11),             // StopInfoUrl
-                        c.getString(12),             // OTP Base URL
-                        c.getString(13)              // OTP Contact Email
+                        c.getInt(9) > 0,            // Supports Embedded Social
```

Comment:
I believe Embedded Social should be moved tot he last index here, as it's the most recently added database field.